ui/page/components/components.go

```diff
@@ -714,3 +733,122 @@ func CoinImageBySymbol(icons *load.Icons, coinName string) *decredmaterial.Image
 	}
 	return nil
 }
+
+func WalletLiveTickets(w *dcrlibwallet.Wallet) ([]dcrlibwallet.Transaction, error) {
+	var tickets []dcrlibwallet.Transaction
+	liveTicketFilters := []int32{dcrlibwallet.TxFilterUnmined, dcrlibwallet.TxFilterImmature, dcrlibwallet.TxFilterLive}
+	for _, filter := range liveTicketFilters {
+		tx, err := w.GetTransactionsRaw(0, 0, filter, true)
+		if err != nil {
+			return nil, err
+		}
+
+		tickets = append(tickets, tx...)
+	}
+
+	return tickets, nil
+}
+
+func StakeToTransactionItems(l *load.Load, txs []dcrlibwallet.Transaction, newestFirst bool, hasFilter func(int32) bool) ([]*TransactionItem, error) {
```

Comment:
Please document this function, don't want to have to read through to understand what it does. It should be a general practice to document all public functions, methods and properties.

Reply:
i didn't author that function, it was originally in the staking utils file(https://github.com/planetdecred/godcr/pull/729/files#diff-645fdc2d09679835c6acde6c7a80862d9375e5eede766ab6775127e8863cd28cL59), i only moved it to components